doc/content/getting-started/user-management/_index.md

```diff
@@ -4,16 +4,26 @@ description: ""
 aliases: [/user-management]
 ---
 
-{{% tts %}} allows network administrators to manage the users that have access to the network. Administrators may add and remove users, and perform any actions on Gateways, End Devices, and Applications.
-
-For non-administrator users, {{% tts %}} manages access rights to Gateways, End Devices, and Applications through collaborations.
-
 This section explains how to manage users and organizations, and how to grant users and organizations rights as collaborators of entities.
 
 <!--more-->
 
+## Administrator Rights
+
+{{% tts %}} allows network administrators to manage the users that have access to the network. Administrators may:
+
+- Manage users in the tenant
+- Manage the tenant configuration
+- Have all rights on all entities in the tenants (or most rights, depending on [Admin Rights Options]({{< ref "/reference/configuration/identity-server#admin-rights-options" >}})
+- Search for all entities in the tenant (if they have rights for all entities)
+- Manage some admin-only fields of OAuth clients and users
+- Permanently delete (purge) entities (instead of just marking them as deleted)
+- Add and remove users, and perform any actions on Gateways, End Devices, and Applications
```

Comment:
Isn't this already covered by "Manage users" and "Have all rights on all entities"?